### DISCUSSION TITLE
Versions and references for frequently changing projects

### DISCUSSION THREAD

**Discussion by fatagun** [dotnet/aspnetcore#23999] [General]
What are the differences between
    
    <ItemGroup>
		<PackageReference Include="SomePackage" />
	</ItemGroup>

and 

    <ItemGroup>
		<PackageReference Include="SomePackage" Version="4.0.0" />
	</ItemGroup>

and

    <ItemGroup>
		<PackageReference Include="SomePackage" Version="4.0.0-*" />
	</ItemGroup>
?

Similarly, 

    <FrameworkReference Include="Microsoft.AspNetCore.App" />

and 
   <FrameworkReference Include="Microsoft.AspNetCore.App" Version="3.1.5" />

In an environment where package versions change very often, what is the strategy to reduce changes in  projects?

Can you please elaborate?

Thanks.

**Comment by Pilchie** [comment]
This is a better discussion for either `NuGet/Home`, or `dotnet/sdk`.
